java/Ice/context/slice/Greeter.ice

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+[["java:package:com.zeroc.demos"]]
```

Comment:
We should use the new `java:identifier` metadata, something like:

```
["java:identifier:com.zeroc.demos.visitor_center"]
```

Also,  file-metadata should never be "attached" to a module.

```